pilot/pkg/networking/util/util.go

```diff
@@ -179,3 +179,15 @@ func ConvertGogoDurationToDuration(d *types.Duration) time.Duration {
 	}
 	return dur
 }
+
+// TranslateTime converts time protos.
+func TranslateTime(in *types.Duration) *time.Duration {
```

Comment:
i mindlessly copied this function not noticing that there's an exact copy above that returns value. agree ptr is not necessary. 
not sure that returning 0 is the right move since that may quietly create problematic timeouts (0 often means wait forever). since you raised this, i can bite the bullet and return and handle error (which will result in some default from the caller) if you agree with this approach? 
i've also left a TODO to move these functions out somewhere in a future PR.

Reply:
Yeah, I think handling the error is better in this case, if we can't use the 0 value gracefully (and yeah, a 0 timeout is dangerous in a lot of situations).